engine/xml-config/src/main/resources/configuration.xsd

```diff
@@ -124,6 +128,33 @@
 		</sequence>
 	</complexType>
 
+    <complexType name="descriptorProvidersType">
+        <choice minOccurs="1" maxOccurs="unbounded">
+            <element name="custom-class" type="ab:customElementType" minOccurs="1" maxOccurs="1">
+                <annotation>
+                    <documentation>
+                        Defines a custom component by a class name and properties values
+                    </documentation>
+                </annotation>
+            </element>
+            <element name="classpath-scanner" type="ab:classpathScannerType" minOccurs="0" maxOccurs="1">
+                <annotation>
+                    <documentation>
+                        Defines which java packages should be scanned for
+                        components to use in AnalyzerBeans.
```

Comment:
Oh, I realize this is a copy-paste, but actually we should replace "AnalyzerBeans" with "DataCleaner" in this file.

FYI - AnalyzerBeans was the old name used for the "engine" of DataCleaner. Now it is fully apart of DC.